lib/elasticband/filter/range.rb

```diff
@@ -0,0 +1,29 @@
+module Elasticband
+  class Filter
+    class Range < Filter
+      attr_accessor :field, :ranges
+
+      RANGES = %i(gt gteq lt lteq time_zone)
+
+      def initialize(field, ranges)
+        self.field = field.to_sym
+        self.ranges = permitted_ranges(ranges)
+      end
+
+      def to_h
+        { range: { field => parsed_ranges } }
+      end
+
+      def permitted_ranges(ranges)
+        ranges.keep_if { |key, _| RANGES.include?(key) }
+      end
+
+      def parsed_ranges
+        ranges[:gte] = ranges.delete(:gteq) if ranges.key?(:gteq)
```

Comment:
I like this way... I think the idea here is to translate, so modifying the own hash sound not a problem to me